.changeset/tall-chicken-repeat.md

```diff
@@ -2,4 +2,4 @@
 '@shopify/polaris': major
 ---
 
-Removed Summer Editions experimental styles and code for the following components: `Avatar`, `AccountConnection`, `ActionList`, `ActionMenu`, `Autocomplete`
+Removed Summer Editions experimental styles and code for the following components: `Avatar`, `AccountConnection`, `ActionList`, `ActionMenu`, `Autocomplete`, `Breadcrumbs`, `Text`
```

Comment:
I think?

```suggestion
Removed Summer Editions experimental styles and code for the following components: `Avatar`, `AccountConnection`, `ActionList`, `ActionMenu`, `Autocomplete`, `Breadcrumbs`
```

Reply:
I added `Text` because I noticed Text changes were merged into `next` but the changeset wasn't updated so figured I might as well update it in this PR 😅